.github/CONTRIBUTING.md

```diff
@@ -81,6 +99,16 @@ $ make html
 
 The built documentation can then be found in `docs/_build/html/`.
 
+To file a pull request, create a new branch on top of the upstream repository:
+
+```console
+git checkout -b my_topical_branch upstream/main
+```
+
+Make your changes, push them to your fork (the remote *origin*) and publish the PR!
+
+To sync the *attrs* repository with your local copy, just pull from the *upstream* remote.
```

Comment:
Note that `git pull` is a combination of `git fetch` and `git merge`. This may confuse people who are not proficient with Git. I'd go for recommending to always run `git fetch --all` before starting to work on a new PR. I think this would result in less confusion and advanced users would do whatever they want to anyway.